.github/workflows/test.yaml

```diff
@@ -115,9 +123,42 @@ jobs:
         test "$system_daily" == "$current_daily"
       shell: bash
 
+  required-static-checks:
```

Comment:
Could you please add a comment explaining why this step is needed?

Reply:
Done